taxi/views.py

```diff
@@ -1,3 +1,18 @@
+from django.http import HttpRequest, HttpResponse
 from django.shortcuts import render
 
-# Create your views here.
+from taxi.models import Driver, Manufacturer, Car
+
+
+def index(request: HttpRequest) -> HttpResponse:
+    num_drivers = Driver.objects.count()
+    num_manufacturers = Manufacturer.objects.count()
+    num_cars = Car.objects.count()
```

Comment:
To avoid unnecessary variable declaration, please place execution of count method it directly into context